packages/mds-compliance/tests/api.spec.ts

```diff
@@ -799,14 +810,12 @@ describe('Tests Compliance API:', () => {
         telemetry: [...telemetry_a, ...telemetry_b]
       }
       Promise.all([db.initialize(), cache.initialize()]).then(() => {
-        Promise.all([cache.seed(seedData), db.seed(seedData)]).then(() => {
-          db.writeGeography({ name: 'la', geography_id: GEOGRAPHY_UUID, geography_json: la_city_boundary }).then(() => {
-            db.writePolicy(COUNT_POLICY_JSON_4).then(() => {
-              db.publishPolicy(COUNT_POLICY_JSON_4.policy_id).then(() => {
-                done()
-              })
-            })
-          })
+        Promise.all([cache.seed(seedData), db.seed(seedData)]).then(async () => {
+          await db.writeGeography({ name: 'la', geography_id: GEOGRAPHY_UUID, geography_json: la_city_boundary })
```

Comment:
can we copy `la_city_boundary` from `ladot_service_areas` into `mds-test-data`?  `ladot_service_areas` is on the chopping block.

Reply:
@karcass Yup. @janedotx let's keep this PR open until my service-area purging PR lands, then it'll be much easier for you to implement the requested change :)

Reply:
This should be actionable now @janedotx